testgrid/specs/deploy.yaml

```diff
@@ -164,6 +164,9 @@
       version: 1.26.x
     flannel:
       version: latest
+    ekco:
+      version: latest
+  flags: "ha"
```

Comment:
```suggestion
  flags: "ekco-enable-internal-load-balancer"
```

Reply:
ha works in the automation, and is much more likely to be specified by the end user IMO